node/cli/src/command.rs

```diff
@@ -812,6 +812,22 @@ pub fn run() -> Result<()> {
 				info!("Parachain id: {:?}", id);
 				info!("Parachain Account: {}", parachain_account);
 				info!("Parachain genesis state: {}", genesis_state);
+				info!(
+					"Is collating: {}",
+					if config.role.is_authority() {
+						"yes"
+					} else {
+						"no"
+					}
+				);
```

Comment:
This is one of my favorite use cases for the ternary operator:

```suggestion
				info!("Is collating: {}", config.role.is_authority() ? "yes" : "no");

```